test/unit/expressions.spec.ts

```diff
@@ -249,7 +249,7 @@ export class ExpressionTests {
     @TestCase("inst.field | 3", 8 | 3)
     @TestCase("inst.field << 3", 8 << 3)
     @TestCase("inst.field >> 1", 8 >> 1)
-    @TestCase("inst.field = 3", 7)
```

Comment:
Is this test right? We should verify if typescript returns the literal number or the field getter.

Reply:
This was previously returning the wrong value. When an assignment is evaluated, the value on the right is returned. The left should not be evaluated. [Playground](https://www.typescriptlang.org/play/#src=class%20Foo%20%7B%0D%0A%20%20%20%20get%20prop()%20%7B%20return%201%3B%20%7D%0D%0A%20%20%20%20set%20prop(val%3A%20number)%20%7B%7D%0D%0A%7D%0D%0Aconst%20foo%20%3D%20new%20Foo()%3B%0D%0Aalert(foo.prop%20%3D%207)%3B%0D%0A)